package.json

```diff
@@ -66,7 +63,9 @@
       "radix": 0,
       "new-cap": 0,
       "no-return-assign": 0,
-      "eol-last": 0
+      "eol-last": 0,
+      "no-restricted-syntax": 0,
+      "guard-for-in": 0
```

Comment:
Probably a good idea to keep `guard-for-in` so you don't get any fun prototype shenanigans in our `for (x in y)` loops